### PR TITLE
Madoko markup fixes, including one stale cross-ref

### PR DESCRIPTION
**Pull request by jafingerhut**

### CODE REVIEW

File: p4-16/spec/P4-16-draft-spec.mdk [hunk 1]
```diff
@@ -3085,8 +3085,8 @@ tuple<bit<32>, bool> x = { 10, false };
 
 ## Operations on sets { #sec-set-exprs }
 
-Some P4 expressions denote sets of values (```set<T>```, for some type
-```T```; see Section [#sec-synth-types].1). These expressions can
+Some P4 expressions denote sets of values (```set<T>```, for some type ```T```;
+see Section [#sec-synth-types].1). These expressions can
```
Comment (jnfoster) [inline p4-16/spec/P4-16-draft-spec.mdk]: Is the .1 intended?

File: p4-16/spec/P4-16-draft-spec.mdk [hunk 2]
```diff
@@ -4681,7 +4681,7 @@ It is illegal to instantiate a ```parser``` within a ```control```
 block.
 For a description of the ```optConstructorParameters```, which
 can be used to build parameterized control blocks, see Section
-[#sec-parametrization].
+[#sec-parameterization].
```
Comment (jnfoster) [inline p4-16/spec/P4-16-draft-spec.mdk]: Weird. I did a search-and-replace for this typo. Someone must have manually backed it out.

File: p4-16/spec/P4-16-draft-spec.mdk (hunk 1 above)
Comment (jafingerhut) [inline p4-16/spec/P4-16-draft-spec.mdk]: I did not add that, but it appears someone wanted to make a cross-ref to the first sub-section of #sec-synth-types, which is named "Set types".   I can try to make a named 'anchor' for that sub-section and see if Madoko likes it.

Comment (jafingerhut) [inline p4-16/spec/P4-16-draft-spec.mdk]: It worked fine.  One less manual cross-ref in the document.  I noticed you had been fixing those as you found them.

**Comment by jnfoster** [comment]
Looks good. Thanks.

**Pull request merged**